src/components/daydream-controls.js

```diff
@@ -18,14 +16,16 @@ var GAMEPAD_ID_PREFIX = 'Daydream Controller';
  */
 module.exports.Component = registerComponent('daydream-controls', {
   schema: {
-    hand: {default: DEFAULT_HANDEDNESS}, // This informs the degenerate arm model.
+    hand: {default: ''}, // This informs the degenerate arm model.
```

Comment:
that is entirely not the case.  the system will have the daydream controller show up with hand 'left' or 'right', so of course apps can use a left-handed or right-handed model as they see fit?!?